golang.req

```diff
@@ -21,6 +33,9 @@ end
 requires = []
 
 unless filelist.empty? then
+
+	requires << "golang(API) = " + go_get_version()
```

Comment:
Hi all. I just hope I can help a bit here... so the problem is that requiring _golang(API) = VERSION_ is needed when packaging .a files , but it is a problem if we are packaging an executable (meaning it has a main function).

So, what we need is basically a way to distinguish the 2 cases.

Right?